scripts/update_java_archives.sh

```diff
@@ -39,20 +39,13 @@ function generate_java_archives() {
 cat << EOM
 "repositories for java"
 
-load("//private/remote:temurin_archive.bzl", "temurin_archive")
+#VERSION ${plain_version}
 
-JAVA_RELEASE_VERSIONS = {
-    "temurin21_jre_amd64": "${plain_version}",
-    "temurin21_jdk_amd64": "${plain_version}",
-    "temurin21_jre_arm64": "${plain_version}",
-    "temurin21_jdk_arm64": "${plain_version}",
-    "temurin21_jre_ppc64le": "${plain_version}",
-    "temurin21_jdk_ppc64le": "${plain_version}",
-}
-
-def repositories():
-    "java archives"
+java = use_extension("//private/extensions:java.bzl", "java")
 EOM
+  exit 1
+
+
```

Comment:
this looks like a debug exit that was left in here?

Reply:
oh yes...